openstack/identity/v3/projects/requests.go

```diff
@@ -0,0 +1,103 @@
+package projects
+
+import (
+	"github.com/rackspace/gophercloud"
+	"github.com/rackspace/gophercloud/pagination"
+)
+
+type project struct {
+	DomainID    string `json:"domain_id"`
+	ParentID    string `json:"parent_id"`
+	Enabled     bool   `json:"enabled"`
+	Name        string `json:"name"`
+	Description string `json:"description"`
+}
+
+// ProjectOpts Options for project create & update
+type ProjectOpts struct {
+	DomainID    string
+	ParentID    string
+	Name        string
+	Enabled     bool
+	Description string
+}
+
+// Create Creates project
+func Create(client *gophercloud.ServiceClient, opts ProjectOpts) CreateResult {
```

Comment:
Instead of having a `struct` parameter here, let's use an `interface` like the other services. For an example, see here: https://github.com/rackspace/gophercloud/blob/master/openstack/networking/v2/networks/requests.go#L127